tests/helpers/test_selector.py

```diff
@@ -359,7 +359,26 @@ def test_text_selector_schema(schema, valid_selections, invalid_selections):
                 ]
             },
             ("red", "green"),
-            ("cat", 0, None),
+            ("cat", 0, None, ["red"]),
+        ),
+        (
+            {"options": ["red", "green", "blue"], "multiple": True},
+            (["red"], ["green", "blue"]),
```

Comment:
The implementation allows an empty list, let's test it:
```suggestion
            (["red"], ["green", "blue"], []),
```